Core/clim-basic/recording.lisp

```diff
@@ -1604,13 +1595,16 @@ were added."
                toward-y transform-glyphs transformation)
       record
     (let ((medium (sheet-medium stream)))
-      (medium-draw-text* medium string point-x point-y start end align-x
-                         align-y toward-x toward-y transform-glyphs))))
+      (with-drawing-options (medium :transformation transformation)
+        (medium-draw-text* medium string point-x point-y start end align-x
+                           align-y toward-x toward-y transform-glyphs)))))
 
 (defrecord-predicate draw-text-output-record
     (string start end point-x point-y align-x align-y toward-x toward-y transform-glyphs)
   (and (if-supplied (string)
          (string= (slot-value record 'string) string))
+       (transformation-equal (graphics-state-transformation record)
+                             (graphics-state-transformation record))
```

Comment:
is there even a remote possibility, that this won't be T? we are calling transformation-equal on the same object after all.